firebase.json

```diff
@@ -3,26 +3,13 @@
     {
       "source": "functions",
       "codebase": "default",
-      "ignore": [
-        "node_modules",
-        ".git",
-        "firebase-debug.log",
-        "firebase-debug.*.log",
-        "*.local"
-      ],
-      "predeploy": [
-        "npm --prefix \"$RESOURCE_DIR\" run lint",
-        "npm --prefix \"$RESOURCE_DIR\" run build"
-      ]
+      "ignore": ["node_modules", ".git", "firebase-debug.log", "firebase-debug.*.log", "*.local"],
+      "predeploy": ["npm --prefix \"$RESOURCE_DIR\" run lint", "npm --prefix \"$RESOURCE_DIR\" run build"]
```

Comment:
Does this eliminate the need for running the build command in ci? If so, how does it take into account build modes?

Reply:
This is a part of the typescript template. I didn't add it with intention. It does appear to call the build script. It doesn't look like modes are forwarded. I should probably remove this since it might be double building. Since wri only uses firebase for previews I haven't noticed or paid much attention.